person.rb

```diff
@@ -0,0 +1,24 @@
+class Person
+  attr_accessor :name, :age, :parent_permission
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
```

Comment:
## Getter for id:
- Please add this code "@id = Random.rand(1..1000)" to generate random numbers